.github/workflows/ci_test.yml

```diff
@@ -0,0 +1,80 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event_name }}
+  cancel-in-progress: true
+
+env:
+  RUST_BACKTRACE: 1
+
+jobs:
+  test:
+    runs-on: ${{ matrix.setup.os }}
+    strategy:
+      matrix:
+        toolchain: ["nightly", "beta"]
```

Comment:
Since beta is the next stable predictably, perhaps we can verify beta directly and announce compio as always compatible with `stable`?

Reply:
At that time AFIT is not stablized. Now compio should be compatible with stable.

Reply:
Got it. File https://github.com/compio-rs/compio/pull/297